src/replication.c

```diff
@@ -1324,20 +1373,43 @@ check_for_command_completion(spec_t *spec, rcommon_cmd_t *rcomm_cmd, ISTGT_LU_CM
 		}
 	}
 
+	response_recieved = success + failure;
+	min_response = MAX(spec->replication_factor -
```

Comment:
we should be taking these from rcommon_cmd structure

Reply:
ok